tanker_identity/test/test_identity.py

```diff
@@ -0,0 +1,79 @@
+import base64
+
+import tanker_identity.crypto
+import tanker_identity.identity
+
+import pytest
+from tanker_identity.test.helpers import corrupt_buffer, check_user_secret, check_signature, parse_b64_json
+
+
+def generate_test_identity(test_trustchain):
+    user_id = "guido@tanker.io"
+    b64_identity = tanker_identity.identity.create_identity(
+        test_trustchain["id"],
+        test_trustchain["private_key"],
+        user_id
+    )
+    identity = parse_b64_json(b64_identity)
+    return identity
+
+
+def test_generate_identity_happy(test_trustchain):
+    identity = generate_test_identity(test_trustchain)
+    delegation_signature = base64.b64decode(identity["delegation_signature"])
+
+    assert identity["trustchain_id"] == test_trustchain["id"].decode()
```

Comment:
not your fault, but the test_trustchain should contain only strings, no bytes

Reply:
Since you offered to off-list, I'd be happy to let you do that =]